### ISSUE TITLE
Error during processing webhook

### ISSUE THREAD

**Issue by alexwesemann** [adnanh/webhook#640]
Good afternoon,
Currently I am running Webhook version 2.6.9 on a virtualized Ubuntu 11 machine. Maybe this is a really noob question, but I am struggling now for a couple of days and running out of inspiration.

My hooks.json file looks like this:
![hooks](https://github.com/adnanh/webhook/assets/74415724/b434b40b-f2d6-46ad-b509-dc8cbaa3434f)
My redeploy.sh script looks like this:
![image](https://github.com/adnanh/webhook/assets/74415724/1026fd53-5d64-4225-8417-8fdf94979a26)

I constantly get the following error when running the listener:
![image](https://github.com/adnanh/webhook/assets/74415724/7f50e449-a267-4b53-bfde-ad6f02f51038)

Can someone tell me what I am doing wrong?
Hope to hear from you guys soon.

**Comment by adnanh** [comment]
Hmm, seems to work with 2.8.0

**Comment by alexwesemann** [comment]
Good morning,

I suppose you mean what's coming in from the KPN IOT platform (I use two
destinations: webhook.site for testing and webhook for production which is
now work in progress). This is what it looks like in Webhook.site:
[image: image.png]
[image: image.png]
Hope this helps?
Let me know if you need more information.
KR
Alex

Op ma 22 mei 2023 om 18:14 schreef Cameron Moore ***@***.***>:

**Comment by alexwesemann** [comment]
Dear Adnan,
Is there an easy way to upgrade to 2.8.0? I am currently on 2.9.6 and
installed a couple of days ago using sudo apt install in debian11.

If the upgrade solves the problem, then I can move on. Basically, the goal
here is to throw every received webhook into the MySQL database and use it
in combination with google maps presenting the measured IOT data to the
user using the latitude and longitude coordinates.

I have the database in place (working with Webhook.site) and the google
maps web application retrieving the data from the database. So, basically
the only step I have to take is transfer the webhook data into the MySQL
database when the bash script is triggered.

Does that make sense?
KR
Alex

Op ma 22 mei 2023 om 22:53 schreef Adnan Hajdarević <
***@***.***>:

**Comment by alexwesemann** [comment]
As an addition to my last e-mail, this is what is coming with the received
webhook:
[{"bn":"urn:dev:IMEI:86***********4:","bt":1.684833304E9},{"n":"CPU
temperatuur","u":"Cel","v":43.312},{"n":"Processorgebruik","u":"%","v":22.5},{"n":"Breedtegraad","u":"lat","v":51.87695},{"n":"Lengtegraad","u":"lon","v":4.61525}]

Op ma 22 mei 2023 om 18:14 schreef Cameron Moore ***@***.***>:

**Comment by alexwesemann** [comment]
Gents,

After a lot of struggling I managed to install and run a 2.8.0 instance of
webhook. This seems to solve the issue. I am now receiving this:
[image: image.png]
But, the milion dollar noob question is: How do I get this data in my
database? Can I use the id and run a php script to insert the data into the
MySQL database like I do with webhook.site? Or is there a more
simpler/common way to do this? Sorry, I need a little bit more information
to point me in the right direction, here.

But, I am already happy that I got this far! :)
KR
Alex

Op ma 22 mei 2023 om 22:53 schreef Adnan Hajdarević <
***@***.***>:

**Comment by alexwesemann** [comment]
Morning, any news on this question?
KR
Alex

Op di 23 mei 2023 om 12:02 schreef Alex Wesemann ***@***.***>:

**Comment by alexwesemann** [comment]
Hi Cameron,Thanks a lot for your explanation. Today, I learned this and changed my script to pass both header and payload data to a python script that extracts the iot information and imports it into the mysql database. There is still a lot of work to do but I think I made some good steps forward today.Appreciate your help and support!Verstuurd vanaf mijn iPhone - Sent from my iPhone,Met vriendelijke groet,Alex WesemannOp 25 mei 2023 om 20:42 heeft Cameron Moore ***@***.***> het volgende geschreven:﻿
I think you're misunderstanding what webhook does.  webhook parses a web request and passes parts of that request to a local script or executable.  It's up to the script or executable to do something useful with what's passed to it.
Your config is telling webhook to pass a JSON representation of the entire headers and payload to your script, but your script isn't doing anything with the parameters that are passed to it.  It's only appending stuff to a log file.
